tensorflow/lite/toco/tflite/op_version.cc

```diff
@@ -66,6 +66,7 @@ std::string GetMinimumRuntimeVersionForModel(const Model& model) {
           {{OperatorType::kBatchToSpaceND, 1}, "1.6.0"},
           {{OperatorType::kBatchToSpaceND, 2}, "1.14.0"},
           {{OperatorType::kBatchMatMul, 1}, kPendingReleaseOpVersion},
+          {{OperatorType::kBatchMatMul, 3}, kPendingReleaseOpVersion},
```

Comment:
The version 2 of the operator seems to be missing but it's present in `runtime_version.cc`. Should I add it in this PR?

The `kPendingReleaseOpVersion` also seems to be outdated compared to `runtime_version.cc`.

Reply:
Line 30 here seems to indicate that no additional updates are required to this file. I would revert this change. The change in `tensorflow/lite/tools/versioning/op_version.cc` seems sufficient. thanks!

Reply:
Thanks I saw the comment on line 30 but I wasn't completely sure if we still needed to update it as it may still be used by TOCO when experimental_new_converter is manually set to false in the TFLiteConverter. I removed the line.

Reply:
OK great, thanks! I'll now try to get this submitted.

Reply:
In the end we don't need to update this file at all any more when bumping the version of an operator? I did it for the following merged PR https://github.com/tensorflow/tensorflow/pull/41487, but I will update my other non-merged PRs to remove the update of `tensorflow/lite/toco/tflite/op_version.cc` then.